notification/twilio/alertsms.go

```diff
@@ -19,31 +20,29 @@ import (
 // then be 70 or 67 characters for single or multi-segmented messages, respectively.
 const maxGSMLen = 160
 
-type alertSMS struct {
-	ID    int
-	Count int
-	Body  string
-	Link  string
-	Code  int
-}
+var alertTempl = template.Must(template.New("alertSMS").Parse(`Alert #{{.AlertID}}: {{.Summary}}
 
-var smsTmpl = template.Must(template.New("alertSMS").Parse(`
-{{- if .ID}}Alert #{{.ID}}: {{.Body}}
-{{- else if .Count}}Svc '{{.Body}}': {{.Count}} unacked alert{{if gt .Count 1}}s{{end}}
-{{- end}}
 {{- if .Link }}
 
 {{.Link}}
-{{- end}}
-{{- if and .Count .ID }}
+{{end}}
 
-{{.Count}} other alert{{if gt .Count 1}}s have{{else}} has{{end}} been updated.
-{{- end}}
 {{- if .Code}}
+Reply '{{.Code}}a' to ack, '{{.Code}}c' to close.{{end}}`))
+
+var bundleTempl = template.Must(template.New("alertBundleSMS").Parse(`Svc '{{.ServiceName}}': {{.Count}} unacked alert{{if gt .Count 1}}s{{end}}
 
-Reply '{{.Code}}a{{if .Count}}a{{end}}' to ack{{if .Count}} all{{end}}, '{{.Code}}c{{if .Count}}c{{end}}' to close{{if .Count}} all{{end}}.
-{{- end}}`,
-))
+{{- if .Link }}
+
+	{{.Link}}
```

Comment:
A lot of the tests require the messages to have extra returns (`\n`) or tabs (`\t`). Do we want to retain this?

Reply:
😢